package.json

```diff
@@ -1,83 +1,32 @@
 {
-  "name": "docusign",
-  "version": "8.0.0",
-  "description": "A DocuSign API helper library with promise support",
-  "main": "docusign.js",
+  "name": "docusign-esign",
+  "version": "2.0.0",
+  "description": "DocuSign Node.js API client.",
+  "license": "LicenseRef-LICENSE",
```

Comment:
Hey @mmallis87 , yes we should be using MIT license for all our clients moving forward...

Thanks,
-Ergin